### ISSUE TITLE
check box value

### ISSUE THREAD

**Issue by anandchawla** [volosoft/jtable#868]
Hi

I want to have a textfield residence address and other as permanent address...and one checkbox as (same as residence)...

on clicking the checkbox...the permanent address should get populated with residence addres.....i am not able to achieve this in jtable...

Please advise me how can i do this in jtable page..all my efforts have failed...

---

however in a test script it seems to work as follows

<!DOCTYPE html>
<html>
<head>

<script src="http://ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js">
</script>

<script>
$(document).ready(function(){
  //$("mylist").change(function(){
    //$("#mytext").hide();
    
  //});
  alert ($("#oldtext").val());
  $('#mybox').change(function()
  {
  var n = $('#mybox').val();
  if($(this).is(":checked"))
  {
     $('#mytext').val($('#oldtext').val());
  }
  else
  {
       $('#mytext').val(" ");
  }
  }
  );
});
</script>

</head>

<body>

<h2>This is a heading</h2>

<p>This is a paragraph.</p>

<p>This is another paragraph.</p>

<p>&nbsp;</p>

<form name="form1" method="post" action="">
  <p>
    <label>
    <input name="mybox" type="checkbox" id="mybox" value="yes">
    </label>
</p>
  <p>
    <select name="mylist" id="mylist">
      <option value="No">No</option>
      <option value="Yes">Yes</option>
    </select>
  </p>
  <p>
    <input name="oldtext" type="text" id="oldtext" value="how are u">
</p>
  <p>
    <input type="text" name="mytext" id="mytext">
</p>
</form>

<p>&nbsp;</p>

<button>Click me</button>
</body>
</html>
